frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx

```diff
@@ -388,24 +385,30 @@ export default function OntologyDagView({
         .descendants()
         .find(
           (node) =>
-            node.data.id.split("__").at(0) === cellTypeId && node.data.children
+            (lastNodeClicked
```

Comment:
If lastNodeClicked is set, center on it. Otherwise center on the target cell type